db/models.py

```diff
@@ -50,3 +60,57 @@
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(settings.AUTH_USER_MODEL,
```

Comment:
Ensure that 'settings.AUTH_USER_MODEL' is correctly configured in your Django settings to point to the appropriate user model.